src/gt4py/next/iterator/ir_utils/common_pattern_matcher.py

```diff
@@ -16,15 +16,15 @@
 _Fun = TypeVar("_Fun", bound=itir.Expr)
 
 
-class _IsCall(itir.FunCall, Generic[_Fun]):
+class FunCallTo(itir.FunCall, Generic[_Fun]):
     fun: _Fun
     args: List[itir.Expr]
 
 
-_IsCallToPattern: TypeAlias = _IsCall[itir.SymRef]
+_FunCallToSymRef: TypeAlias = FunCallTo[itir.SymRef]
```

Comment:
I would make all these symbols public, or private if you prefer, but I don't understand why would `FunCallTo` be public while `_FunCallToSymRef` and similar are private...